docs/page-objects/inputs.rst

```diff
@@ -0,0 +1,65 @@
+.. _inputs:
+
+======
+Inputs
+======
+
+:ref:`Page object classes <page-object-classes>`, in their ``__init__`` method,
+must define input parameters with type hints pointing to input classes.
+
+Those input classes may be:
+
+-   :ref:`Built-in web-poet input classes <built-in-inputs>`.
+
+-   :ref:`Custom input classes <custom-inputs>`.
+
+-   Other :ref:`page object classes <page-object-classes>`.
```

Comment:
Technically, it's other Injectable classes. Just a heads up: if you change the example from Injectable to ItemPage in the explanation for page object classes, it may require some changes here.

Reply:
[I’ve added an extra bullet point for “other injectable classes”](https://github.com/scrapinghub/web-poet/pull/152/commits/71ddcdfb760b7e8f9fc732a70e56e1f21aa796cb).